packages/salesforcedx-vscode-apex/test/vscode-integration/requirements.test.ts

```diff
@@ -60,4 +63,45 @@ describe('Java Requirements Test', () => {
     expect(requirements.java_home).contains(jdk);
   });
 
+  it('Should support Java 8', async () => {
+    execFileStub.yields('', '', 'build 1.8.0');
+    checkJavaVersion('~/java_home')
```

Comment:
Looks like previous tests in this file at least are using the 
``` 
try { 
  await call
  assert
} catch () { 
``` flow. 
should we stay consistent?

Reply:
My approach is usually to use .then() and .catch(), as it feels more lower-level to me, but since the other tests use await, I updated these tests to use try/await/catch as well.